proto/data.go

```diff
@@ -636,6 +636,8 @@ func (ms *MVCCStats) Add(oms *MVCCStats) {
 	ms.IntentCount += oms.IntentCount
 	ms.IntentAge += oms.IntentAge
 	ms.GCBytesAge += oms.GCBytesAge
+	ms.SysBytes += oms.SysBytes
+	ms.SysCount += oms.SysCount
 	ms.LastUpdateNanos += oms.LastUpdateNanos
```

Comment:
We're currently treating the update nanos as a delta when we call this add method, so it's being used correctly.